tests/conftest.py

```diff
@@ -136,8 +140,22 @@ def _make_client(auth: Tuple[str, str]) -> AsyncClient:
 
 
 @pytest.fixture(autouse=True)
-def flush_server(server: str):
-    assert requests.post(f"{server}/__flush__")
+async def flush_default_collection(
+    make_client: ClientFactory,
+    auth: Auth,
+    source_bucket: str,
+    source_collection: str,
+):
+    yield
+    client = make_client(auth)
+
+    try:
+        await client.delete_collection(
+            id=source_collection, bucket=source_bucket, if_exists=True
+        )
+    except KintoException:
+        # in the case where a user doesn't have permissions to delete
+        pass
```

Comment:
In the heartbeat test, no user is ever created, so no bucket/collection/permissions are created either. In this teardown phase, that user will not have permissions to tear down the collection, but we don't want that to fail the test. I could print the error here. What do you think?

Reply:
Ok, I get it now. If just the heartbeat will produce a `print()` then yes!

What we want to avoid is the integration tests to fail without a clue about it because the user don't have the appropriate perms on the collection